examples/BackupRestoreTags/tagutil.go

```diff
@@ -0,0 +1,176 @@
+package main
```

Comment:
Should we just call this file `main.go`? The `BackupRestoreTags` folder seems to name the tool?